modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -445,3 +446,7 @@
     (when-not (str/blank? dataset-id)
       (convert-dataset-id-to-filters! database dataset-id))
     database))
+
+(defmethod driver/prettify-native-form :bigquery-cloud-sdk
+  [_ native-form]
+  (sql.u/format-sql-and-fix-params :mysql native-form))
```

Comment:
We could call `(driver/prettify-native-form :mysql native-form)`, right? That would require less intimate knowledge of how pretty printing is implemented.